modules/standard/GMP.chpl

```diff
@@ -560,7 +560,7 @@ module GMP {
     }
     proc set_str(str:string, base:int=0)
     {
-      on this do mpz_set_str(this.mpz, str.c_str(), base.safeCast(c_int));
+      on this do mpz_set_str(this.mpz, str.localize().c_str(), base.safeCast(c_int));
```

Comment:
I'm curious about all these localize calls.  Intuitively, I'm guessing it to make sure that the resulting c string is local to the locale in question.  What happens if they aren't used?  (I'm guessing runtime error only in the case that the string was remote).  What happens without them?  What happened in master prior to this PR?  This might be another topic to add to the list for the team given how frequently it shows up in the internal modules...